generators/server/templates/src/main/java/package/web/rest/errors/_ExceptionTranslator.java

```diff
@@ -88,18 +88,7 @@
     @Override
     public ResponseEntity<Problem> handleMethodArgumentNotValid(MethodArgumentNotValidException ex, @Nonnull NativeWebRequest request) {
         BindingResult result = ex.getBindingResult();
```

Comment:
You can just remove the overriden method completely

Reply:
Right...